style.css

```diff
@@ -0,0 +1,635 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap');
+
+html,
+body {
+  overflow-x: hidden;
+}
+
+body {
+  font-family: "Lato", sans-serif;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
- [ ] Dear @monicah79, Although you did great following the requested changes, here as the previous reviewer mentioned, I can't find the Cocogoose font family or a similar font Please Fix that.